src/mdlComponentHandler.js

```diff
@@ -51,19 +51,6 @@ var componentHandler = (function() {
   }
 
   /**
-   * Returns true if the given element has already been upgraded for the given
-   * class.
-   * @param {HTMLElement} element The element we want to check.
-   * @param {string} jsClass The class to check for.
-   * @return boolean
-   * @private
-   */
-  function isElementUpgraded_(element, jsClass) {
```

Comment:
@Garbee 
Because it was not used often; only twice in the previous code. If this method was kept, it would almost only contain:
```JavaScript
return upgradedList.indexOf(optJsClass) === -1;
```
And passing upgradedList in by the caller would make this function unnecessary to me; it almost only replaces `Array.indexOf` but it adds extra function calls.
If instead generating `upgradedList` inside this method, that's some extra data fetching and string splitting to slow things down.